libs/community/langchain_community/vectorstores/__init__.py

```diff
@@ -27,6 +27,9 @@
         VectorStore,
```

Comment:
**Static Code Issue**: Move import 'langchain_core.vectorstores.VectorStore' out of type-checking block. Import is used for more than type hinting. <br> **Fix**: Move the import statement outside the TYPE_CHECKING block to the top-level imports. <br> **Code Suggestion**: 
 ```
 - if TYPE_CHECKING:
 -     from langchain_core.vectorstores import (
 -         VectorStore,
 -     )
 + from langchain_core.vectorstores import VectorStore
 ```